CODEOWNERS

```diff
@@ -223,7 +223,7 @@ homeassistant/components/keba/* @dannerph
 homeassistant/components/keenetic_ndms2/* @foxel
 homeassistant/components/kef/* @basnijholt
 homeassistant/components/keyboard_remote/* @bendavid
-homeassistant/components/knx/* @Julius2342 @farmio @marvin-w
+homeassistant/components/knx/* @XKNX/xknx-devs
```

Comment:
I don't think that our bot, which is not a member of XKNX, is able to tag that team.

Reply:
Ah okay, that's unfortunate - I'll just change it back then :). Thanks